integration/pull_test.go

```diff
@@ -112,7 +114,6 @@ func TestOptimizeConsistentSociArtifact(t *testing.T) {
 			containerImage: "alpine:latest",
 		},
 	}
-	const blobStorePath = "/var/lib/soci-snapshotter-grpc/content/blobs/sha256"
```

Comment:
Where does this come from now?

Reply:
https://github.com/awslabs/soci-snapshotter/blob/main/integration/create_test.go#L32